data/src/main/java/com/architect/coders/mu8/data/comics/ComicsDAO.kt

```diff
@@ -17,6 +17,9 @@ interface ComicsDAO {
     @Insert(onConflict = IGNORE)
     fun insertComics(comics: List<ComicsEntity>)
 
-    @Query("SELECT characters_id FROM comics_table WHERE id = :comicId" )
-    fun getComicCharacters(comicId : Long) : List<String>
+    @Query("SELECT characters_id FROM comics_table WHERE id = :comicId")
+    fun getComicCharacters(comicId: Long): List<String>
```

Comment:
What is this query used for? Can you explain?

Reply:
was already created in issue-32, look develop